tests/Microsoft.Graph.DotnetCore.Core.Test/Serialization/SerializerTests.cs

```diff
@@ -339,6 +340,38 @@ public void SerializeDateValue()
             Assert.Equal(expectedSerializedString, serializedString);
         }
 
+        [Fact]
+        public void SerializeDateTimeOffsetValue()
+        {
+            // Arrange
+            var dateTimeOffset = DateTimeOffset.Parse("2016-11-20T18:23:45.9356913+00:00");
```

Comment:
nit: you should also have unit tests with offsets, our CI runs on machines configured on UTC, which often leads to false negatives

Reply:
Resolved via https://github.com/microsoftgraph/msgraph-sdk-dotnet-core/pull/234/commits/a50d4a6b08a8e0944b7474f094b21b0fe314b8c8